docs/pt/mkdocs.yml

```diff
@@ -67,6 +67,8 @@ nav:
   - tutorial/extra-data-types.md
   - tutorial/query-params-str-validations.md
   - tutorial/cookie-params.md
+  - tutorial/header-params.md
```

Comment:
Nice job with the translation!
Why have you added this line to `mkdocs.yml`? As far as I know the header params page is not translated yet hence it should not be added to this file yet

Reply:
sorry, I added it by mistake.